crates/chain/src/chain_graph.rs

```diff
@@ -27,7 +27,7 @@ use core::fmt::Debug;
 #[derive(Clone, Debug, PartialEq)]
 pub struct ChainGraph<P = TxHeight> {
     chain: SparseChain<P>,
-    graph: TxGraph,
+    graph: TxGraph<BlockId>,
```

Comment:
this should just stay as `TxGraph` where the default it used (which should be `()`).

Reply:
Done in 67a4abae85b1bdaf16092a753f5f2a5f8726d769